### PR TITLE
dockerize helloworld-api

### PR DESCRIPTION
**Pull request by spectroalias**
api dockerized and running.

### CODE REVIEW

File: djangoProject/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+asgiref==3.4.1
+Django==3.2.9
+djangorestframework==3.12.4
+pytz==2021.3
+sqlparse==0.4.2
```
Comment (tucosaurus) [inline djangoProject/requirements.txt:5]: Why is this needed here ? This should be a part of PR #2

**Review by tucosaurus — CHANGES_REQUESTED** [review]
Remove requirement file additions

File: djangoProject/requirements.txt (hunk 1 above)
Comment (spectroalias) [reply djangoProject/requirements.txt:5]: @tucosaurus i made a seperate PR #2  for api and only related to api as these were not needed in DRF project api-part so i add these on dockerize-part PR #3  as they were needed here.
